icons/evry.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>EVRY icon</title><path d="M.443 12.919c.102.255.113.426.072.692-.03.202-.068.404-.108.605-.035.174-.117.55.003.694.117.139.35.13.622.098.302-.037 1.81-.342 3.397-.997.215-.093.435-.205.622-.345.431-.324.5-1.005.5-1.005.011-.13.01-.316-.164-.413-.126-.07-.3-.026-.562.043-.433.116-1.502.519-1.873.628-.15.044-.447.114-.56-.027-.19-.234.081-.97.244-1.378.163-.41.42-.738 1.205-.96.29-.082.762-.23.954-.293.211-.071.49-.207.607-.675.135-.534-.278-.616-.655-.718-.539-.147-.878-.165-.64-.68.096-.21.186-.302.396-.357.214-.056 2.03-.338 2.751-.467.405-.072.542-.286.711-.631.365-.746-.093-.794-.462-.761-.836.076-2.178.315-2.753.397-.306.044-.102-.47-.46-.494-.704-.05-1.302.243-1.54.425-.227.172-.335.703-.335.703L1.718 9.37c-.06.273-.146.543-.343.659-.479.283-1.457.302-1.37 1.34 0 0 .02.51.437 1.55zm6.318-4.424c.28-.047.654-.115.756.586.031.212.032 2.66.032 2.66s.27-.251 1.094-1.525C9.27 9.25 9.96 8.005 10.22 7.568c.152-.256.433-.522.918-.165.541.398.553.685.315 1.093-.244.419-.9 1.486-1.827 2.772-.842 1.168-1.245 1.63-1.599 2.016-.354.385-.826.83-1.319.762-.654-.091-.604-1.198-.63-1.515-.026-.317-.154-2.832-.066-3.42.034-.227.025-.495.75-.617zm8.845.065c.302.16-.045.855-.626 1.277-.374.272-1.014.679-1.672.679a6.206 6.206 0 011.017-1.298c.47-.461 1.018-.8 1.281-.658zm8.17-.153c.29-.373.349-.596-.133-1.061-.433-.417-.723-.171-.934.045-.035.042-1.016 1-2.049 2.17-.804.91-1.499 1.756-1.499 1.756s-.025-.57.062-1.286c.049-.408.186-1.201.226-1.453.204-1.274-.593-1.325-.89-1.343-.312-.02-.46.145-.554.533-.085.356-.55 2.31-.365 4.325.033.363.105.737.112.76l-.188.258c-1.749-.485-2.946-1.664-2.946-1.664s.574-.165 1.204-.543c.501-.3 1.465-1.043 1.363-2.353-.087-1.112-1.27-1.62-1.707-1.53-.397.084-1.09.559-1.747 1.14-.044-.303-.052-.638-.368-.675-.232-.027-.5.082-.696.182a1.116 1.116 0 00-.5.473l-.737 1.665c-.024.059-.072.072-.11.08-.394.09-.52.37-.549.449 0 0-.165.347.126.66-.146.386-.533 1.283-.533 1.283-.48 1.1-.449 1.483-.277 1.717.087.119.393.29.762.35.278.047.491-.118.723-.481.343-.537 1.032-1.953 1.032-1.953s1.37 1.474 4.121 2.443c0 0-1.455 1.973-1.665 3.064 0 0-.108.715.872.713.246 0 .617-.058.886-.221a.475.475 0 00.223-.281.216.216 0 00-.02-.153c-.028-.047-.07-.073-.102-.104-.05-.05-.043-.113-.016-.187.54-1.544 2.732-4.17 3.197-4.738 1.233-1.507 3.512-3.854 3.673-4.041z"/></svg>
```

Comment:
Looks like some differences between PR (red) and source (black) - would you mind rebuilding this one?
![image](https://user-images.githubusercontent.com/1830867/102893672-e46c1a00-4459-11eb-9095-ff7c3ea9a105.png)